posthog/cdp/templates/make/template_make.py

```diff
@@ -0,0 +1,58 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="free",
+    type="destination",
+    id="template-make",
+    name="Make",
+    description="Sends a webhook templated by the incoming event data",
```

Comment:
Seems a bit generic?

Reply:
Changed the description to `Triggers a webhook based scenario`